pkg/operator/k8sutil/prometheus_test.go

```diff
@@ -32,6 +32,7 @@ func TestGetServiceMonitor(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "rook-ceph-mgr", servicemonitor.GetName())
 	assert.Equal(t, "rook-ceph", servicemonitor.GetNamespace())
+	assert.NotNil(t, servicemonitor.GetLabels())
```

Comment:
we test this for `PrometheusRule`, I think it's just good to test the same here, too